lib/vehicle.rb

```diff
@@ -22,4 +24,18 @@ def antique?
   def electric_vehicle?
     @engine == :ev
   end
+
+  def new_registration_date
+     @registration_date = Date.today
+  end
+
+  def plate_type
+  if electric_vehicle?
+    :ev
+  elsif Date.today.to_s[0, 4].to_i - @year < 25
+    :regular
+    else
+      :antique
+    end
+  end
```

Comment:
1. keep an eye on your indentation
2. With the methods `.electric_vehicle?` and `.antique?` available, I think you could make this conditional more DRY and readable:
```
def plate_type
  if electric_vehicle?
    :ev
  elsif antique?
    :antique
  else
    :regular
  end
end
```